ReleaseProcessAutomation/Jira/JiraVersionCreator.cs

```diff
@@ -70,4 +70,16 @@ public string CreateNewVersionWithVersionNumber (string versionNumber)
 
     return createdVersionID;
   }
+
+  public JiraProjectVersion? FindVersionWithVersionNumber (string versionNumber)
+  {
+    var versions = _projectVersionFinder.FindVersions(_config.Jira.JiraProjectKey, "(?s).*").ToList();
```

Comment:
General questions: Why do we need the `(?s)` in our regex?

Reply:
That is a great question. Tested it with and without it, no difference. 
The received data consists of individual objects already anyways. Maybe theres a possibility that the string is too long for jira and it cuts it off beforehand, but even with long names like 2.2.0-alpha.5 that didnt happen so... Possibly redundant, but it doesnt really hurt us and gives some safety against whatever happens in Jira.

Reply:
I guess that. `(?s)` is a non-capturing group with the `s` being an option for the group, but the group is empty. So it never matches anything and as a non-capturing group it won't create a group so there is no difference and we should remove it.

`Now the only thing left is `.*` which just matches everything. So we should remove the filter entirely to make it more readable and have an `.FindVersions(projectKey)` overload.

Reply:
FYI: it was introduced here, but there's no reason for the single-line option:
https://github.com/re-motion/BuildTools/commit/a0424ddb530e9639c22baf86fd83ace21fa1ba1d#diff-e27d394fb9ff4d75b627934eec2ad5c69b6682139219f019f72209620ab4559bR54